### PR TITLE
Changing GitHub Package Registry to GitHub Container Registry

### PR DESCRIPTION
**Pull request by anais-codefresh**
* Updating the docs on GitHub Docker Registry to showcase the use of GitHub Container Registry
* Replacing the naming of GitHub Docker Registry to GitHub Container Registry across the documentation

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/anais-github-container-registry/docs/

File: _docs/integrations/docker-registries/github-container-registry.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+---
+title: "GitHub Container Registry"
+description: "Learn how to push Docker images to your GitHub Container Registry"
+group: integrations
+redirect_from:
+  - /docs/integrations/docker-registries/github-packages/
+toc: true
+---
+
+[GitHub Packages](https://github.com/features/packages) is an artifact storage solution that among other formats, can also be used as a Docker registry.
```
Comment (kostis-codefresh) [inline _docs/integrations/docker-registries/github-container-registry.md:10]: Remove all mentions of Github packages. They will disappear as soon as Github registry gets out of beta. So the documentation page should be simplified to just mention Github registry.

File: _docs/integrations/docker-registries/github-container-registry.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    stage: "build"
+    registry: "github-container-registry"
+{% endraw %}
+{% endhighlight %}
+
+Notice:
+
+* The `registry: github-container-registry` property in the `build` step, which is the name of the registry that you set-up in the previous step
+* The fact that we push multiple Docker tags in a single step; you can define all tags in the `build` step
+* The prefix `anais-codefresh/react-article-display` so that the final image is compliant with GitHub naming.
```
Comment (kostis-codefresh) [inline _docs/integrations/docker-registries/github-container-registry.md:173]: That's exactly whey we have the "repository prefix" field in the registry settings. So in the pipeline users should not have to do something extra. Or at least this is how it should work...

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/anais-github-container-registry/docs/

File: _docs/integrations/docker-registries/github-container-registry.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+You can make sure that your token is valid by using it as password on your local workstation with the docker command:
+
+```
+docker login ghcr.io --username github-account
+[Paste your GitHub token on this prompt]
+```
+
+**Important** Make sure that the URL is correct, otherwise, you will receive login errors later on. The github-account is your GitHub username.
+
+### Tag and push your Docker image
```
Comment (kostis-codefresh) [inline _docs/integrations/docker-registries/github-container-registry.md:49]: Write a short intro here that says that is this the MANUAL way of using the Github registry and that in the next section we will completely automate all this with Codefresh.

This way readers will understand that they can use Codefresh and Github registry right away from the UI. They don't have to use the docker CLI in advance

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/anais-github-container-registry/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/anais-github-container-registry/docs/

**Pull request merged**